crates/ironrdp-rdpdr-native/src/nix/simple.rs

```diff
@@ -0,0 +1,765 @@
+use ironrdp_pdu::PduResult;
+use ironrdp_svc::{impl_as_any, SvcMessage};
+use libc::ftruncate;
+use nix::libc::{closedir, opendir, readdir, DIR};
+
+use ironrdp_rdpdr::pdu::efs::*;
+use ironrdp_rdpdr::pdu::esc::{ScardCall, ScardIoCtlCode};
+use ironrdp_rdpdr::pdu::RdpdrPdu;
+use ironrdp_rdpdr::RdpdrBackend;
+use std::ffi::{CStr, CString};
+use std::io::{Seek, SeekFrom, Write};
+use std::os::fd::{AsFd, AsRawFd};
+use std::os::unix::fs::MetadataExt;
+use tracing::{debug, warn};
```

Comment:
suggestion: Put this in lib.rs so we don’t need to `use` the logging macros. Usually I don’t recommend to do that expect for tracing.

```rust
#[macro_use]
extern crate tracing;
```

Reply:
ok,this seems good. i modify the code